build.sh

```diff
@@ -39,7 +39,7 @@ fi
 detect_platform && set_build_vars
 
 function check_not_overriding {
-    if [ -e "$CONDA_PREFIX"/lib/python*/site-packages/"$1".egg-link ]; then
+    if [[ -e "$CONDA_PREFIX"/lib/python*/site-packages/"$1".egg-link ]]; then
```

Comment:
This actually needs to use single braces, otherwise it doesn't work
```
~/Desktop/temp> ls
a.txt
~/Desktop/temp> if [ -e *.txt ]; then echo "exists"; else echo "not"; fi
exists
~/Desktop/temp> if [[ -e *.txt ]]; then echo "exists"; else echo "not"; fi
not
```
Of course, neither of them works properly if there can be more than one file matching the pattern.
```
~/Desktop/temp> touch b.txt
~/Desktop/temp> if [ -e *.txt ]; then echo "exists"; else echo "not"; fi
-bash: [: a.txt: binary operator expected
not
~/Desktop/temp> if [[ -e *.txt ]]; then echo "exists"; else echo "not"; fi
not
```